agent/agent-tooling/build.gradle.kts

```diff
@@ -26,8 +26,6 @@ dependencies {
   implementation("net.bytebuddy:byte-buddy")
 
   implementation("commons-codec:commons-codec")
-  implementation("org.apache.commons:commons-lang3")
```

Comment:
the standalone exporter probably shouldn't have any dependencies on apache commons to avoid version conflict for users (this is not a problem in the agent where we hide all of our transitive dependencies from users)